src/domain/synthetics/orders/simulateExecuteTxn.tsx

```diff
@@ -100,33 +123,27 @@ export async function simulateExecuteTxn(chainId: number, p: SimulateExecutePara
   const tenderlyConfig = getTenderlyConfig();
 
   if (tenderlyConfig) {
-    await simulateTxWithTenderly(
-      chainId,
-      exchangeRouter as BaseContract,
-      p.account,
-      "multicall",
-      [simulationPayloadData],
-      {
-        value: p.value,
-        comment: `calling ${method}`,
-      }
-    );
+    const router = method === "simulateExecuteGlvDeposit" ? glvRouter : exchangeRouter;
```

Comment:
also for simulateExecuteGlvWithdrawal

Reply:
maybe move this logic above, so that `router` is always used instead of glvRouter/exchangeRouter